### PR TITLE
Fix minor code issues in 5g branch as part of review.

### PR DESCRIPTION
**Pull request by heplesser**
@jakobj Issues mostly addressed here are
- include ordering
- repeated creation of dictionaries inside loops
- some compiler warning (g++ 8)

### CODE REVIEW

File: models/rate_transformer_node_impl.h [hunk 1]
```diff
@@ -66,7 +66,6 @@ template < class TNonlinearities >
 nest::rate_transformer_node< TNonlinearities >::Parameters_::Parameters_()
   : linear_summation_( true )
 {
-  recordablesMap_.create();
```
Comment (jakobj) [inline models/rate_transformer_node_impl.h]: why was this removed?

File: libnestutil/config.h.in [hunk 2]
```diff
@@ -88,6 +88,9 @@
 /* libneurosim support enabled? */
 #cmakedefine HAVE_LIBNEUROSIM 1
 
+/* Define to 1 if you have usable long long type. */
+#cmakedefine HAVE_LONG_LONG 1
+
```
Comment (jakobj) [inline libnestutil/config.h.in]: why was this added?

File: models/rate_transformer_node_impl.h (hunk 1 above)
Comment (heplesser) [reply models/rate_transformer_node_impl.h]: I removed it here because it is also called in the constructor for the neuron itself, and that is the canonical place to call it.
